peps/pep-9999.rst

```diff
@@ -157,36 +157,406 @@ splitting off headers and other build-time dependencies in a separate
 this could be made both more comprehensive and easier to maintain though a tool
 command with semantics of *"show the system package manager install command for
 all external dependencies"*. This may be done as a standalone tool, or as a new
-command in any Python development workflow tool (e.g., Pip, Poetry, Hatch, PDM).
+subcommand in any Python development workflow tool (e.g., Pip, Poetry, Hatch, PDM).
 
 
 Registry design
 ---------------
 
-- One central Python package for the registry tool
-- A plugin design, where system package managers can register their name mapping
-- Name mapping should be in a structured, human-readable file format (TBD:
-  JSON, YAML, or ...)
+The mapping infrastructure should include the following components and properties:
+
+- A central registry for known generic and virtual PURLs.
```

Comment:
Does this need rephrasing to use `dep:`? I guess we need a good name for that. They're not actually PURLs anymore. Any thoughts or preferences?

Reply:
Oh yes, good catch, I was still using "PURL" everywhere. Changed all the PURL occurrences to "`dep:` identifier" for now. If we can come up with something better, I'm happy to change it, but I think it's sufficiently descriptive. The term `dep:` identifier is first introduced in the first mention of PEP 725 external dependencies, so I hope is sufficiently clear.

Reply:
Yep, LGTM now, thanks. 

> The term `dep:` identifier is first introduced in the first mention of PEP 725 external dependencies, so I hope is sufficiently clear.

It's clear to us, we can deal with further clarifications at public review time if needed. Resolving this comment.